scripts/ipfs/ipfs-swarm-cjdns.sh

```diff
@@ -33,4 +33,4 @@ while read -r cjdns_peer; do
 done <<< `sudo nodejs /opt/cjdns/tools/peerStats 2>/dev/null | awk '{ if ($2 == "ESTABLISHED") print $1 }' | awk -F. '{ print $6".k" }' | xargs`
 
 # Update peers data since ipfs just started
-/usr/local/bin/nodeinfo-update.sh
+sudo /usr/local/bin/nodeinfo-update.sh
```

Comment:
Does this mean that future nodeinfo modules will need to work under sudo? Like because this sudo will execute `run-parts` with sudo, which will execute all the nodeinfo files (just IPFS for now) with sudo. The IPFS script just curls from the 5001 api, but if it used just the `ipfs id` command then this sudo would have made it fail, if my thinking is correct. Would this be a problem in the future, where scripts might need to not use sudo?

Reply:
The other places that this runs are all run as root there are also some permission issues that's creep up if you don't run it as sudo although they can be mitigated with it running as root in other places anyway I figured this was the shortest path and somewhat standardized

Reply:
On issue i found is basically 
https://github.com/tomeshnet/prototype-cjdns-pi/blob/da224ec2f36d985c3de859829af8c21a0d74eafb/scripts/shared/nodeinfo/install#L12
Copies over a file with root write perms and
https://github.com/tomeshnet/prototype-cjdns-pi/blob/da224ec2f36d985c3de859829af8c21a0d74eafb/scripts/shared/nodeinfo/nodeinfo-update.sh#L3
Tries to write it but it cant unless its sudoed